pkg/sloop/processing/eventcount.go

```diff
@@ -69,6 +70,15 @@ func updateEventCountTable(
 
 	eventCountByMinute := spreadOutEvents(computedFirstTs, computedLastTs, computedCount)
 
+	if involvedObject.Uid == "" {
```

Comment:
can involvedObject be nil here? Are we receiving only involvedObject without Uid here?

Reply:
No Its already checked here https://github.com/salesforce/sloop/blob/master/pkg/sloop/processing/processing.go#L56 Yes we are receiving some involved objects with Uid empty